crates/nix_rs/src/copy.rs

```diff
@@ -0,0 +1,19 @@
+use crate::command::{CommandError, NixCmd};
+use std::path::PathBuf;
+
+/// Runs `nix copy` in Rust
+pub async fn from_nix(cmd: &NixCmd, host: &str, paths: Vec<&PathBuf>) -> Result<(), CommandError> {
```

Comment:
Prefer `&[PathBuf]` over mandating the user to build `Vec` every time.

Reply:
Also, `from_nix` is only used when reading a certain Rust type from Nix. `nix copy` on the other hand does no reading. This should be called `nix_copy`